docs/Serverless.md

```diff
@@ -25,6 +25,7 @@ choice with an additional snippet of code.
 - [Google Cloud Run](#google-cloud-run)
 - [Netlify Lambda](#netlify-lambda)
 - [Vercel](#vercel)
+- [Google Cloud Functions](#google-cloud-functions)
```

Comment:
How are the two Google things different? Regardless, let's alphabetize the list (and order in documentation).

Reply:
I agree. Thank you @jsumners